src/config.ts

```diff
@@ -34,16 +39,18 @@ function filterAndTypecastEnvs (env: any) {
     API_PORT,
     CACHE_ENABLED,
     HASURA_URI,
+    PROMETHEUS_METRICS,
     QUERY_DEPTH_LIMIT,
     TRACING
   } = env
   return {
     allowedOrigins: ALLOWED_ORIGINS,
     apiPort: Number(API_PORT),
-    cacheEnabled: Boolean(CACHE_ENABLED),
+    cacheEnabled: CACHE_ENABLED === 'true',
     hasuraUri: HASURA_URI,
+    prometheusMetrics: PROMETHEUS_METRICS === 'true',
     queryDepthLimit: Number(QUERY_DEPTH_LIMIT),
-    tracing: Boolean(TRACING)
+    tracing: TRACING === 'true'
```

Comment:
I at first thought there had to be a better way to handle env var strings with nodejs so it properly handles yes/no, true/false, 1/0, etc... but apparently there isn't :sob: https://github.com/lorenwest/node-config/issues/272#issuecomment-270005104

Abstracted in nix as a boolean so at least nix users won't :cry:.

Reply:
Right! The initial implementation was actually a bug, as it would cast `TRACING=false` to `true` :dizzy_face: